mscs

```diff
@@ -1965,7 +1979,7 @@ case "$1" in
     if serverRunning "$2"; then
       # If the world server has users logged in, announce that the world is
       # being deleted.
-      if [ $(printf "%d" $(queryStatus $2 | cut -f6)) -gt 0 ]; then
+      if [ $(queryNumUsers $WORLD) -gt 0 ]; then
```

Comment:
Hmm, I didn't look very closely before accepting this patch.  Shouldn't this line be:
```
      if [ $(queryNumUsers $2) -gt 0 ]; then
```